### ISSUE TITLE
Admin Layout refactoring

### ISSUE THREAD

**Issue by romaninsh** [atk4/ui#137]
Currently Admin Layout is crude and not responsive. It needs to be re-designed by someone with better CSS experience.

``` php
include 'vendor/autoload.php';

echo str_repeat("this text will be obstructed", 20);

$app = new \atk4\ui\App('hello');
$app->initLayout('Admin');
$app->layout->add('LoremIpsum');
```

Result:

<img width="1075" alt="screen shot 2017-04-19 at 11 51 48" src="https://cloud.githubusercontent.com/assets/453929/25176611/9fb3caba-24f6-11e7-8858-18b3bc951960.png">

**Comment by DarkSide666** [comment]
This issue is not because Admin Layout (which of course is not the best design-wise), but because `echo` will echo text before App is even rendered. As result you will have HTML like this:
```
this text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructedthis text will be obstructed<!DOCTYPE html>
<html lang="en">
  <head>
    <title>hello</title>
    ....
```
That of course will not render properly.

My suggestion is to use output buffering `ob_start()` and App then can catch that and inject it inside it's Layout Content tag to generate proper HTML structure.

**Comment by romaninsh** [comment]
it's possible to do that without output buffering too.

**Comment by romaninsh** [comment]
fixed in #170
